src/app/(protected)/[group]/[subGroup]/[instance]/(admin-panel)/(stage-6)/allocation-overview/_components/by-project-columns.tsx

```diff
@@ -28,6 +28,10 @@ export interface ProjectData {
   userId: string;
 }
 
+// TODO: fix procedure so that data model and UI are not tightly coupled
```

Comment:
Created Issue about DataTables, maybe I should move all of the DataTable related TODOs there? What do you think  @AlysonDick ?